libs/partners/milvus/langchain_milvus/vectorstores/milvus.py

```diff
@@ -298,8 +319,30 @@ def __init__(
         self._primary_field = primary_field
         # In order for compatibility, the text field will need to be called "text"
         self._text_field = text_field
+
+        if isinstance(self.embedding_func, list):
+            if len(self.embedding_func) < 2:
```

Comment:
Didn't see the whole PR yet, so just wondering: is there a real problem with the implementation if you send a list of length 1? If it works, I think it will be better to allow it.

Reply:
It is a real problem since the hybrod retriever will not allow that: https://github.com/langchain-ai/langchain/blob/f66b7ba32da6e008589247d04f871585d909a7e4/libs/partners/milvus/langchain_milvus/retrievers/milvus_hybrid_search.py#L74

I take alternative approach - if of length 1, extract it to a field.
Do you think it's better?

Reply:
mmm, in that case I'm not sure. I think I would still prefer extracting it and allowing that.